src/maggma/stores/mongolike.py

```diff
@@ -750,7 +766,7 @@ def update_json_file(self):
             data = [d for d in self.query()]
             for d in data:
                 d.pop("_id")
-            json.dump(data, f)
+            json.dump(data, f, default=json_serial)
```

Comment:
Might make sense to use ORJSON here. It can natively serialize datetime objects and would be faster. It is also already a dependency of maggma.

Reply:
Thanks, done! I was not familiar with `orjson` but it looks like a good move.